src/mapeo-project.js

```diff
@@ -206,51 +207,61 @@ export class MapeoProject extends TypedEmitter {
         dataStore: this.#dataStores.data,
         table: observationTable,
         db,
+        translation: this.$translation,
```

Comment:
In my testing, `this.$translation` is `undefined` at this point. I think we need a way to avoid this circular reference.

Reply:
With your merged PR this would be solved, right?

Reply:
Yup!